src/System.Memory/tests/Span/CopyTo.cs

```diff
@@ -34,6 +34,24 @@ public static void TryCopyToSingle()
         }
 
         [Fact]
+        public static void TryCopyToDefaultImplicit()
+        {
+            int[] dst = default;
+            Span<int> srcSpan = default;
+            bool success = srcSpan.TryCopyTo(dst);
+            Assert.True(success);
+        }
+
+        [Fact]
+        public static void TryCopyToNullImplicit()
+        {
+            int[] dst = null;
+            Span<int> srcSpan = default;
+            bool success = srcSpan.TryCopyTo(dst);
```

Comment:
Why call a method rather than just doing the cast?

Reply:
Because I want to test the implicit conversion from a null array to a Span.

Reply:
Right.  Why does that require calling a method?

Reply:
Nvm, I see what you mean.